distribution/lib/Standard/Base/0.0.0-dev/src/Data/Numbers.enso

```diff
@@ -975,13 +975,15 @@ type Integer
 
             Integer.parse "20220216"
     parse : Text -> Text -> Integer ! Parse_Error
-    parse text (radix=10) =
-        Panic.catch NumberFormatException (Long.parseLong text radix) _->
-             Error.throw (Number_Parse_Error.Error text)
+    parse text (radix=10) = Integer.parse_internal text radix
+
+    parse_internal text radix = @Builtin_Method "Integer.parse"
```

Comment:
Should there be a `PRIVATE` modifier explaining that users should use `parse` like in the other similar cases? See date-related builtins.

Reply:
+1 for `PRIVATE`

Reply:
Marked private at 7c5bcc600